packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -295,14 +293,16 @@ describe('IndexedDb: canActAsPrimary', () => {
     return;
   }
 
-  async function clearOwner(): Promise<void> {
+  async function clearPrimaryLease(): Promise<void> {
     const simpleDb = await SimpleDb.openOrCreate(
       INDEXEDDB_TEST_DATABASE,
       SCHEMA_VERSION,
       createOrUpgradeDb
     );
-    await simpleDb.runTransaction('readwrite', [DbOwner.store], txn => {
-      const ownerStore = txn.store<DbOwnerKey, DbOwner>(DbOwner.store);
+    await simpleDb.runTransaction('readwrite', [DbPrimaryClient.store], txn => {
+      const ownerStore = txn.store<DbPrimaryClientKey, DbPrimaryClient>(
+        DbPrimaryClient.store
+      );
       return ownerStore.delete('owner');
```

Comment:
How about creating `DbPrimaryClient.key` similar to how we have `DbTargetGlobal.key`?